ArticleTemplates/assets/build/components/require.js.src.js

```diff
@@ -0,0 +1,2076 @@
+/** vim: et:ts=4:sw=4:sts=4
```

Comment:
is this someone else's code? should it be coming from npm instead of checked in?